sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 
 ### Bugs Fixed
 
+- Fixed a bug where the client was not retrying when a connection drop happened ([34786](https://github.com/Azure/azure-sdk-for-python/pull/34786))
+- Fixed a bug where the client would not handle a role instance swap on the service correctly ([34820](https://github.com/Azure/azure-sdk-for-python/pull/34820))
+
 ### Other Changes
 
```

Comment:
Could we add this here too?
```suggestion
### Other Changes
 
- Updated the logging to more accurately represent when frames are being sent to prevent a client-side idle timeout ([#34793](https://github.com/Azure/azure-sdk-for-python/pull/34793)).
 
```